packages/@lightningjs/ui-components/src/components/Icon/Icon.js

```diff
@@ -149,10 +149,9 @@ export default class Icon extends Base {
         src: this.icon
       };
     }
-    const supportsColor = !Boolean(svgTag || svgURI);
     const color = getValidColor(this._color || this.style.color);
 
-    if (!supportsColor && color) {
+    if (color) {
       context.warn('icon does not allow colors to be applied to an SVG');
```

Comment:
should this error message change or be removed? im not sure what case we're catching here

Reply:
🤦‍♀️ yes, removing, thank you! Didn't read what the if statement was doing